keanu-project/src/main/java/io/improbable/keanu/algorithms/mcmc/NUTS.java

```diff
@@ -65,6 +65,13 @@ public static NUTS withDefaultConfig(KeanuRandom random) {
     @Builder.Default
     private double targetAcceptanceProb = DEFAULT_TARGET_ACCEPTANCE_PROB;
 
+    //The target acceptance probability, a suggested value of this is 0.65,
+    //Beskos et al., 2010; Neal, 2011
```

Comment:
I think this comment applies to the item above?  Adding a comment explaining what the tree height is doing would be awesome though.